include/zephyr/net/wifi_mgmt.h

```diff
@@ -462,6 +462,22 @@ struct wifi_connect_req_params {
 	const uint8_t *key_passwd;
 	/** Private key passwd length */
 	uint8_t key_passwd_length; /* Max 128 */
+	/** private key2 passwd */
+	const uint8_t *key2_passwd;
+	/** key2 passwd length */
+	uint8_t key2_passwd_length; /* Max 128 */
+	/** suiteb or suiteb-192 */
+	uint8_t suiteb_type;
+	/** eap version */
+	uint8_t eap_ver;
+	/** Identity for EAP */
+	const uint8_t *eap_identity;
+	/** eap identity length */
+	uint8_t eap_id_length; /* Max 64 */
+	/** Password string for EAP. */
+	const uint8_t *eap_password;
+	/** eap passwd length */
+	uint8_t eap_passwd_length; /* Max 128 */
```

Comment:
all the length comments can be clubbed to the main comment for them to come up in doxygen.

Reply:
Updated as comment.